### PR TITLE
Config for sphinx astra nationalstrassenachsen

### PR DESCRIPTION
**Pull request by Sgachet**

### CODE REVIEW

**Comment by pfanguin** [comment]
Config chsdi : https://github.com/geoadmin/mf-chsdi3/pull/2244

**Comment by pfanguin** [comment]
https://github.com/geoadmin/service-sphinxsearch/pull/299

File: conf/uvek.conf.part [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## SOURCES
+# SOURCES
```
Comment (gjn) [inline conf/uvek.conf.part]: Why this change?

Comment (pfanguin) [inline conf/uvek.conf.part]: Just an error.
